doc/nrf/index.rst

```diff
@@ -10,7 +10,10 @@ The |NCS| includes the Zephyr™ real-time operating system (RTOS), which is bui
 
 .. note::
 
-   This version of the |NCS| supports prototyping and evaluation, but should not be used for production and deployment in end products.
+   nRF53 Series devices are now supported for production.
+   Bluetooth mesh is supported for development in |release| for prototyping and evaluation.
```

Comment:
Should "mesh" be removed to make it a generic statement about Bluetooth support. I believe all of Bluetooth is supported for prototyping and evaluation. @omkar3141 ?

Reply:
This is specifically related to Mesh, as it is not yet ready for production support. I would say mesh could be kept here and remove "for prototyping and evaluation" (cliche for development). There could be a separate statement for other Bluetooth features if needed.

Reply:
Maybe my point wasn't clear. Bluetooth in general is only supported for development on nRF53 as of now. I consider Bluetooth mesh a subset of this.

I see @jdstapes requested to change the statement to:

> Wireless protocol stacks and libraries may indicate support for development or support for production for different series of devices based on verification and certification status.

This is fine from SoftDevice controller's point of view.